hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java

```diff
@@ -127,9 +127,10 @@ public RecordStreamManager(
 					startWriteAtCompleteWindow,
 					RecordStreamType.RECORD);
 			writeQueueThread = new QueueThreadObjectStreamConfiguration<RecordStreamObject>()
-					.setThreadName("writeQueueThread")
 					.setNodeId(platform.getSelfId().getId())
+					.setCapacity(nodeLocalProperties.recordStreamQueueCapacity())
```

Comment:
this `.setCapacity` is newly added, should have a test to cover this line. The same for the (new) line 146.